Mathlib/Analysis/Convex/Strict.lean

```diff
@@ -28,23 +28,25 @@ open Convex
 
 section OrderedSemiring
 
-variable [OrderedSemiring 𝕜] [TopologicalSpace E] [TopologicalSpace F]
+/-- A set is strictly convex if the open segment between any two distinct points lies is in its
+interior. This basically means "convex and not flat on the boundary". -/
+@[nolint unusedArguments]
+def StrictConvex (𝕜 : Type*) {E : Type*}
+    [Semiring 𝕜] [PartialOrder 𝕜] [IsOrderedRing 𝕜] [TopologicalSpace E]
```

Comment:
Similarly here, rather than adding tehnical debt, we should remove `[IsOrderedRing 𝕜] ` really, maybe add a TODO that there are lots of `omit`s and hopefully someone will come along and clean it up; I think adding all the `omit`s might convince someone it needs doing, whereas nobody will notice if you add technical debt (except now during the review process).